coalib/bearlib/abstractions/Linter.py

```diff
@@ -196,6 +197,11 @@ def check_prerequisites(cls):
                         check_call(options["prerequisite_check_command"],
                                    stdout=DEVNULL,
                                    stderr=DEVNULL)
+                        warnings.simplefilter("default")
```

Comment:
coala is used in CI. putting lots of deprecation messages into other ppls CI logs is counter productive. They want clean logs.